src/org/usfirst/frc/team4213/robot/Robot.java

```diff
@@ -8,18 +8,42 @@
 import org.team4213.lib14.CowCamController;
 import org.team4213.lib14.CowCamServer;
 
+import edu.wpi.first.wpilibj.CANTalon;
 import edu.wpi.first.wpilibj.DriverStation;
 /* @Authors:
- * --
- * 
- * 
- * @Mentors
- * --Tim Robert
- * --
+
+
```

Comment:
It was an old comment I had added.  Was going to comment all those that contributed.  We dont need the @authors